### PR TITLE
feat(chat): Add crow to admins in group chat

### PR DESCRIPTION
**Pull request by borismelnik**
Closes: #5897

### What does the PR do

Add crown to admins

### Affected areas

Members in chat view

### Screenshot of functionality

Снимок экрана 2022-05-30 в 12 59 59

### CODE REVIEW

File: ui/app/AppLayouts/Chat/panels/UserListPanel.qml [hunk 1]
```diff
@@ -80,6 +80,14 @@ Item {
             isOnline: model.onlineStatus
             icon.color: Theme.palette.userCustomizationColors[Utils.colorIdForPubkey(model.id)]
             ringSettings.ringSpecModel: Utils.getColorHashAsJson(model.id)
+            components: [
+                StatusIcon {
+                    anchors.verticalCenter: parent.verticalCenter
+                    visible: model.isAdmin
+                    source: Style.svg("crown")
+                    color: Theme.palette.inputColor
```
Comment (PascalPrecht) [inline ui/app/AppLayouts/Chat/panels/UserListPanel.qml:88]: Not sure what `inputColor` is, but if we want black in light mode and white in darkmode, we should probably be using `directColor1` to stay consistent..

Comment (borismelnik) [reply ui/app/AppLayouts/Chat/panels/UserListPanel.qml:88]: that fair :) I will fix

Comment (borismelnik) [reply ui/app/AppLayouts/Chat/panels/UserListPanel.qml]: fixed

**Comment by borismelnik** [comment]
> 

Okay, if not? if we reuse it in future? And it's not the same like `isMutualContact`, `chatKey` and so on. We made this component with possibilities to add somethings at the left side of component and I use it. For example, if I add it to component, Should I use the same way for it? If yes, how it should be done in future if we need to add new icon to the end or middle of component and its should be different in two places? I know how - change implementation and make component bigger and bigger and more complex. And I found that comment in community `            // TODO: use StatusMemberListItem (it does not behave correctly right now)`, and maybe there no crowns for admins? I don't know. Now it used in one place and now I know that crowns should be just in one place. I will agree if you can tell me where is another place for it. Otherwise, I think it's not necessary.

**Comment by PascalPrecht** [comment]
> We made this component with possibilities to add somethings at the left side of component and I use it

So.. it just so happens that `StatusMemberListItem` has this feature because it is a `StatusListItem`. `StatusListItem` is the component that is mean to have this feature of providing slots, so that we can build components like `StatusMemberListItem` which serve a specific purpose (a member list item) which may or may not have a crown.

Now.. this is not to push for my suggestion. I'm fine with not moving it there. I just think that it makes sense given that it's the ... well, `StatusMemberListItem`.

Oh and lastly: of course we can't foresee if there might be another thing that has to be added to that item in specific circumstances... you can then still do what you're already doing here - use the component slots and have full control over what things should be rendered there.

**Comment by borismelnik** [comment]
> > We made this component with possibilities to add somethings at the left side of component and I use it
> 
> So.. it just so happens that `StatusMemberListItem` has this feature because it is a `StatusListItem`. `StatusListItem` is the component that is meant to have this feature of providing slots, so that we can build components like `StatusMemberListItem` which serve a specific purpose (a member list item) which may or may not have a crown.
> 
> Now.. this is not to push for my suggestion. I'm fine with not moving it there. I just think that it makes sense given that it's the ... well, `StatusMemberListItem`.
> 
> Oh and lastly: of course we can't foresee if there might be another thing that has to be added to that item in specific circumstances... you can then still do what you're already doing here - use the component slots and have full control over what things should be rendered there.

Anyway, I understand and thought about it, but I think its more flexible for now.

**Comment by osmaczko** [comment]
> Now it used in one place and now I know that crowns should be just in one place. I will agree if you can tell me where is another place for it. Otherwise, I think it's not necessary.

I think it should be also used in group chat's members dialog:
image

which probably is not `UserListPanel` and still may use `StatusMemberListItem` as a delegate for the list there.

Another argument that should convince you is this screen from the sandbox app, how would you update it to display the crown there?
image

>  And it's not the same like isMutualContact, chatKey and so on.

To me `StatusMemberListItem` is something specific. I can't see how `isAdmin` differs from `isMutualCotact` from abstraction level perspective :thinking:

**Review by osmaczko — APPROVED** [review]

**Review by endulab — APPROVED** [review]

**Comment by borismelnik** [comment]
@osmaczko @PascalPrecht @endulab guys pls mark it as tested)))

**Pull request merged**